assets/js/payment-method-extensions/payment-methods/paypal/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { registerPaymentMethod } from '@woocommerce/blocks-registry';
+import { __ } from '@wordpress/i18n';
+import { getSetting } from '@woocommerce/settings';
+import { decodeEntities } from '@wordpress/html-entities';
+
+/**
+ * Internal dependencies
+ */
+import { PAYMENT_METHOD_NAME } from './constants';
+
+const settings = getSetting( 'paypal_data', {} );
+
+/**
+ * @typedef {import('@woocommerce/type-defs/registered-payment-method-props').RegisteredPaymentMethodProps} RegisteredPaymentMethodProps
+ */
+
+/**
+ * PayPal content component
+ *
+ * @param {RegisteredPaymentMethodProps|Object} props Incoming props
+ */
+const Content = ( { activePaymentMethod } ) => {
+	return activePaymentMethod === PAYMENT_METHOD_NAME ? (
+		<div>{ decodeEntities( settings.description || '' ) }</div>
+	) : null;
+};
+
+const Edit = ( props ) => {
+	return <Content { ...props } />;
+};
+
+const paypalPaymentMethod = {
+	name: PAYMENT_METHOD_NAME,
+	label: (
+		<strong>
+			{ decodeEntities(
+				settings.title || __( 'PayPal', 'woo-gutenberg-products-block' )
+			) }
+		</strong>
+	),
```

Comment:
Mocks have Stripe showing the card icons; should we switch both to image based?

Reply:
In the case of Stripe, I think we already had a convo somewhere (can't recall where offhand - might have been slack) where we switched to text because cc icons were too generic. We haven't had the same convo for paypal yet. However, being that the paypal logo is a bit more specific and identifiable, I think the rationale for cc icons for Stripe CC isn't the same as that for Paypal.

I'm not opposed to moving back to cc icons for stripe though being that we now have context aware icon output.

Reply:
Also, if we're both asking questions about this (I'm on the fence) we maybe should ask for design feedback (cc @garymurray or @LevinMedia ). The question is whether we should follow the mocks for payment method tab labels as per this:

<img width="740" alt="Image 2020-04-27 at 4 44 33 PM" src="https://user-images.githubusercontent.com/1429108/80419074-6fcee480-88a6-11ea-918f-915432b04895.png">

or just use `Credit Card` and `Paypal` text labels. At some point during development I was instructed to change CC cards to Credit Card for the stripe cc payment method (sorry I don't have any context for that decision handy) but at the time I think that was because we didn't have any method implemented for making sure the available CC options were correct (now we do).

If you think we should just use icons/logo, the next question is what we should use for things like `cheque`.

Reply:
I prefer icons where available fwiw :) That does complicate Stripe though because it's icons vary with setup.

Reply:
> That does complicate Stripe though because it's icons vary with setup.

Right, but since `label` is provided by the payment method, stripe can still return what's needed for here. There's some overlap between this issue and the discussion in #2267 in this case.

Reply:
I will get the icon added here so I can add "icons" to the other PR in progress for paypal too. That one might be a custom one, rather than a built in CC, so I can use it as an example use case.